app/geo/gui/src/ts/twxplore/gui/geo/states/map/MapState.ts

```diff
@@ -1,9 +1,11 @@
 import {MapFeature} from "./MapFeature";
-import {MapFilterState} from "./MapFilterState";
+import { filterStateTypes} from "./MapFilterStateTypes";
 
 export interface MapState {
   keplerGlInstanceRegistered: boolean;
   features: MapFeature[];
   typesVisibility: {[index: string]: boolean};
-  featureTypesFilters: {[featureType: string]: MapFilterState};
+  featureTypesFilters: {[featureType: string]: {[attribute: string]: filterStateTypes}};
```

Comment:
You can use numbers as object indices if that's what this is [attribute: number]. I'm not sure.

Reply:
I want this to be a map of maps.
E.g. featureTypesFilters['Transmission']['frequency'] which returns a {min, max, idx} filterStateType object